### PR TITLE
Update todos app tutorial 2

### PR DESCRIPTION
**Pull request by markmhx**
Fixed version of https://github.com/blockstack/docs/pull/939 as rebased against master

### CODE REVIEW

**Comment by markmhx** [comment]
@aulneau deployment now works. I think this is good to merge and release.

File: src/pages/authentication/building-todo-app.md [hunk 1]
```diff
@@ -10,25 +10,25 @@ images:
   sm: /images/pages/todo-app-sm.svg
 ---
 
-# Building a Todo app
+# Learn from Todos app with Stacks
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:13]: I'm not sure I like this language. 
```suggestion
# Learn from a todos app built with Stacks
```

File: src/pages/authentication/building-todo-app.md [hunk 2]
```diff
@@ -66,101 +65,75 @@ To create a production build, use yarn build.
 
 You should see the app's landing page:
 
-![The homepage of the todos app](/images/todos-home.png)
+!["Todos app landing" screen](/images/todos/landing.png)
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:68]:
```suggestion
!["Todos" app landing screen](/images/todos/landing.png)
```

File: src/pages/authentication/building-todo-app.md [hunk 3]
```diff
@@ -66,101 +65,75 @@ To create a production build, use yarn build.
...
 
 ## Onboard into your first Stacks app
 
 ### Step 1: Choose **Get started** to start onboarding into the app.
 
 The app displays a standardized introductory modal using
-[Stacks authentication](https://github.com/blockstack/stacks.js/tree/master/packages/auth)
+Stacks authentication.
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:75]:
```suggestion
Stacks Connect.
```

File: src/pages/authentication/building-todo-app.md [hunk 4]
```diff
@@ -66,101 +65,75 @@ To create a production build, use yarn build.
...
-their private data.
-
-The modal displays the app's name and icon as configured in
-[`src/components/App.jsx`](https://github.com/blockstack/stacks-todos/blob/master/src/components/App.jsx#L26):
-
-```jsx
-// src/components/App.jsx
+// src/auth.js
+
+import { AppConfig, UserSession, showConnect } from '@stacks/auth';
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:84]:
```suggestion
import { AppConfig, UserSession, showConnect } from '@stacks/connect';
```

File: src/pages/authentication/building-todo-app.md [hunk 5]
```diff
@@ -66,101 +65,75 @@ To create a production build, use yarn build.
...
 }
 
+export function getPerson() {
+  return new Person(getUserData().profile);
+}
 ```
 
-This component loads the [`UserSession`](https://blockstack.github.io/stacks.js/classes/usersession.html)
-module from `@stacks/auth`
+The `authenticate` function implements the `showConnect` function imported from the `@stacks/auth` library.
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:114]:
```suggestion
The `authenticate` function implements the `showConnect` function imported from the `@stacks/connect` library.
```

File: src/pages/authentication/building-todo-app.md [hunk 6]
```diff
@@ -176,10 +149,10 @@ componentDidMount() {
 
 ### Step 2: Choose **Get started** to generate a _Secret Key_.
 
-The app triggers a popup window in which [the Stacks App](https://github.com/blockstack/ux/tree/master/packages/app)
-is loaded from [`app.blockstack.org`](http://app.blockstack.org/) and begins generating a new _Secret Key_.
+The app triggers a popup window in which [Stacks Connect](https://github.com/blockstack/ux/tree/master/packages/app)
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:152]:
```suggestion
The app triggers a popup window in which [Stacks Authenticator](https://github.com/blockstack/ux/tree/master/packages/app)
```

File: src/pages/authentication/building-todo-app.md [hunk 7]
```diff
@@ -212,17 +184,21 @@ and associated with your _Secret Key_.
 Once you've authenticated the app, you can can start adding todos by entering values into the "Write your to do"
 field and hitting "Enter".
 
-![The authenticated view of the todos app](/images/todos-home-authenticated.svg)
+!["Todos app home" screen](/images/todos/home.png)
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:187]:
```suggestion
!["Todos" app home screen](/images/todos/home.png)
```

File: src/pages/authentication/building-todo-app.md [hunk 8]
```diff
@@ -264,38 +248,36 @@ The app will now show all of your todos to anyone who visits the URL displayed w
...
 
-This triggers an event, which
-[under the hood](https://github.com/blockstack/stacks-todos/blob/master/src/components/Header.jsx#L47)
-calls the [`signUserOut` method](https://blockstack.github.io/stacks.js/classes/usersession.html#signuserout)
-of the `UserSession` object.
+This calls the [`signUserOut`](https://blockstack.github.io/stacks.js/classes/usersession.html#signuserout) method
+of the `userSession` object within `src/components/Header.jsx`.
 
-Now, visit the URL that was provided to you when you made your tasks public. This url is of the format `/todos/:username`, so if your username is `jane_doe.id.blockstack`, the URL would be [`localhost:3000/todos/jane_doe.id.blockstack`](http://localhost:3000/todos/jane_doe.id.blockstack).
+Now visit the URL that was provided to you when you made your tasks public. This URL has the format `/todos/:username`, so if your username were `janedoe.id.blockstack`, the URL would be `localhost:3000/todos/jane_doe.id.blockstack`.
```
Comment (aulneau) [inline src/pages/authentication/building-todo-app.md:254]:
```suggestion
Now visit the URL that was provided to you when you made your tasks public. This URL has the format `/todos/:username`, so if your username were `janedoe.id.blockstack`, the URL would be `localhost:3000/todos/janedoe.id.blockstack`.
```

**Review by aulneau — CHANGES_REQUESTED** [review]
I've made suggested changes for how things are/will be currently after our discussion today:

- `@stacks/connect` will export functions from `@stacks/auth`
- `app.blockstack.org` is not _currently_ called `Stacks Connect` and as such I've updated the naming to `Stacks Authenticator`. Until we decided on the name we should keep the package connect and the auth app separate.

**Pull request merged**